recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.10.0" %}
+{% set checksum = "9bca41bf9974cff78b9c9b677feacd0c413e79b16cf31ab5d3a3de4e9c48d6db" %}
+{% set checktype = "sha256" %}
+
+package:
+    name: pycairo
+    version: {{ version }}
+
+source:
+    fn: master.zip
+    url: https://github.com/Jactry/pycairo/archive/master.zip
+    {{ checktype }}: {{ checksum }}
+
+build:
+    number: 0
+    skip: true                  # [win or not py34]
+    script:
+        - python setup.py install
```

Comment:
Should have the arguments ` --single-version-externally-managed --record=record.txt` because of `setuptools`.